### DISCUSSION TITLE
[FEATURE REQ] Dismissable banner to alert to IRC Issues

### DISCUSSION THREAD

**Discussion by A69Tor** [autobrr/autobrr#1668] [Ideas]
**Is your feature request related to a problem? Please describe.**
When one/many of your IRC connections experience issues and are set to "unhealthy network" there is no clear indicator on other parts of autobrr to indicate you should look at your IRC health. For example, network X disconnects, fails to NickServ Identify and stays in Unhealthy for several hours. Unless you are actively on the IRC page you will have to wait until you realize you are not getting downloads.

**Describe the solution you'd like**
A dismissable banner at the top of autobrr on all/any page that indicates one or more of your IRC connections is experiencing an issue and you should go to the IRC page. Link directly to the page in the banner if possible, and once the message is dismissed for that instance, it shouldn't re-appear on any pages until another incident occurs.

**Comment by zze0s** [comment]
We have previously discussed internal notifications like the external ones for Discord or Telegram etc but handled in app only.

Would that solve/fix this for you?

**Comment by A69Tor** [comment]
> We have previously discussed internal notifications like the external ones for Discord or Telegram etc but handled in app only.
> 
> Would that solve/fix this for you?

I haven't used the external ones much, but I am guessing it would solve what I am looking for here. So long as we have a proactive notification that and IRC has got into an unhealthy state so we can determine if its the IRC or the Autobrr instance, the internal notifications should work!
